### PR TITLE
docs(getting-started): fix invalid hammerjs import

### PR DESCRIPTION
**Pull request by Splaktar**
improvements to header hierarchy and auto-generated anchors
- new anchor `/guide/getting-started#step-1`
- old anchor `/guide/getting-started#step-1-install-angular-material-npm-packages`
add link to hammerjs appendix entry
builds upon changes in PR #16654

### CODE REVIEW

File: guides/getting-started.md [hunk 1]
```diff
@@ -2,14 +2,14 @@
...
 
 If you are new to Angular or getting started with a new Angular application, see [Angular's full Getting Started Guide](https://angular.io/start) and [Setting up your environment](https://angular.io/guide/setup-local).
 
-For existing applications, follow the steps below to begin using Angular Material:
+### Follow these steps to begin using Angular Material
 
-
-### Step 1: Install Angular Material npm packages
+#### Step 1
+**Install Angular Material NPM packages**
```
Comment (crisbeto) [inline guides/getting-started.md]: Wouldn't the previous approach be better for SEO? Moving it out means that we don't have any keywords in the header anymore. Also changing the anchors like this will mess up any places where we've linked to it until now.

Comment (jelbourn) [reply guides/getting-started.md]: I would prefer to keep this text in the header (here and below)

Comment (Splaktar) [reply guides/getting-started.md]: I don't think that we've linked to these yet because they were changed in PR https://github.com/angular/components/pull/16654/files and those changes have never been published to the docs site yet.

It does help to have those words in the headers for SEO, but it makes for really long anchors (see description).

Can we shorten this to just?
> ### Install Angular Material

Which would be `/guide/getting-started#install-angular-material`.

Does the "step 1" and "step 2" text add any value? Can we remove that?

Comment (Splaktar) [reply guides/getting-started.md]: Updated, PTAL
